### DISCUSSION TITLE
Trying to run rule at system boot, but doesn't work

### DISCUSSION THREAD

**Discussion by ronnyandre** [arendst/Tasmota#19284] [Q&A] [answered]
I have this command that sets Rule1:

`Rule1 on system#boot do RfRaw 166 endon`

The console logs back:

```
14:54:52.006 RUL: Stored uncompressed, would compress from 31 to 22 (-29%)
14:54:52.010 MQT: stat/rf_bridge/RESULT = {"Rule1":{"State":"OFF","Once":"OFF","StopOnError":"OFF","Length":31,"Free":480,"Rules":"on system#boot do RfRaw 1 endon"}}
```

This should set RfRaw to 166 once MQTT and Wifi is up and running.

However, when rebooting the device, it doesn't trigger. Why?

**Comment by sfromis** [comment, marked as answer]
You did not enable the rule set.

**Reply by ronnyandre** [reply]
Thanks, I was really wondering why it didn't work. Feel stupid now.
